plan/logical_plan_test.go

```diff
@@ -822,7 +822,7 @@ func (s *testPlanSuite) TestRefine(c *C) {
 		},
 		{
 			sql:  "select a from t where c not between 1 and 2",
-			best: "Index(t.c_d_e)[[-inf,1) (2,+inf]]->Projection",
+			best: "Index(t.c_d_e)[[-inf <nil>,1 <nil>) (2 +inf,+inf +inf]]->Projection",
```

Comment:
c < 1 for index(c_d_e) shoule lead to range `[-inf <nil>,1 <nil>)` , for avoiding sweep the whole `1`